test/common/common/utility_test.cc

```diff
@@ -116,6 +117,54 @@ TEST(DateUtil, NowToMilliseconds) {
   EXPECT_EQ(12345067, DateUtil::nowToMilliseconds(test_time));
 }
 
+TEST(OutputBufferStream, All) {
```

Comment:
Please separate these test cases. If there is a failure, it's easier to figure out where things are going wrong if the test cases are granular and each address a single behavior.